common/src/main/java/gov/cms/ab2d/common/model/Contract.java

```diff
@@ -73,7 +82,7 @@ public boolean updateAttestation(boolean attested, String attestationDate) {
             return true;
         }
 
-        String dateWithTZ = attestationDate + " 0:0 " + getESTOffset();
+        String dateWithTZ = attestationDate + " " + getESTOffset();
```

Comment:
Do we know that the date provided is in EST?

Reply:
Yes, it is in Eastern and I've asked HPMS to add that fact to their API documentation.